main/solution/backend/config/infra/functions.yml

```diff
@@ -185,3 +185,7 @@ workflowLoopRunner:
     APP_DOMAIN_NAME: ${self:custom.settings.domainName}
     APP_ENV_BOOTSTRAP_BUCKET_NAME: ${self:custom.settings.environmentsBootstrapBucketName}
     APP_CUSTOM_USER_AGENT: ${self:custom.settings.customUserAgent}
+    APP_ENABLE_EGRESS_STORE: ${self:custom.settings.enableEgressStore}
```

Comment:
Didn't see this defined in `defaults` yaml file. Can we add the config there and default it to `false`.

Reply:
Yes, the code change in default file is pushed in another PR [here](https://github.com/awslabs/service-workbench-on-aws/pull/514).